From ef106350d4837d9509a636491c723fc5aae8df96 Mon Sep 17 00:00:00 2001
From: Arun Raghavan <arun@arunraghavan.net>
Date: Fri, 12 Oct 2018 19:20:51 +0530
Subject: [PATCH 64/72] avdemux: Expose IFF container support

This exposes support for the "iff" demuxer. This is a general purpose
format, and the reason to expose this now is to allow reading DSD data
(which is supported via a variant called DSDIFF).

More information at:
  https://wiki.multimedia.cx/index.php?title=IFF
  https://www.loc.gov/preservation/digital/formats/fdd/fdd000245.shtml
---
 ext/libav/gstavdemux.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/ext/libav/gstavdemux.c b/ext/libav/gstavdemux.c
index 9b6d302..30e0360 100644
--- a/ext/libav/gstavdemux.c
+++ b/ext/libav/gstavdemux.c
@@ -2098,7 +2098,8 @@ gst_ffmpegdemux_register (GstPlugin * plugin)
         !strcmp (in_plugin->name, "ivf") ||
         !strcmp (in_plugin->name, "brstm") ||
         !strcmp (in_plugin->name, "bfstm") ||
-        !strcmp (in_plugin->name, "gif") || !strcmp (in_plugin->name, "dsf"))
+        !strcmp (in_plugin->name, "gif") ||
+        !strcmp (in_plugin->name, "dsf") || !strcmp (in_plugin->name, "iff"))
       rank = GST_RANK_MARGINAL;
     else {
       GST_DEBUG ("ignoring %s", in_plugin->name);
-- 
2.17.0

